lib/api/liskApi.js

```diff
@@ -13,6 +13,27 @@
  *
  */
 
+/**
+ * This is the description for the `Lisk API` module.
```

Comment:
Can we give an actual description like other modules? Following the existing pattern in other modules.

> LiskAPI module provides functions for interfacing with the Lisk network. Providing mechanisms for:
> 
> - Retrieval of blockchain data: accounts, blocks, transactions.
> - Local signing of transactions and immediate network transmission.

Something like that.